### PR TITLE
SP-5826 GitHub base & get-file

### PR DESCRIPTION
**Pull request by nshem**

### CODE REVIEW

**Review by oriser — APPROVED** [review]
Nice!
LGTM

**Review by oriser — APPROVED** [review]
LGTM

File: steps/utils/replace-string/manifest.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: stackpulse.io/v1
 kind: Step
 metadata:
   name: "replace_string"
-  description: "This step replaces sub-string by another sub-string for give base64 string."
+  description: "This step replaces sub-string or a regular expression by another sub-string for given base64 string."
```
Comment (hagaishapira) [inline steps/utils/replace-string/manifest.yaml:5]:
```suggestion
  description: "This step replaces sub-string or a regular expression by another sub-string for a given base64 string."
```

File: steps/utils/replace-string/manifest.yaml [hunk 2]
```diff
@@ -18,4 +18,7 @@ envs:
   - name: SUB_STR_DEST
     description: "The sub-string to replace with."
     type: "string"
-    required: true
\ No newline at end of file
+    required: true
+  - name: REGEX
+    type: bool
+    description: "Wheter the sub string source (SUB_STR_SRC) is a regular expression to match against."
```
Comment (hagaishapira) [inline steps/utils/replace-string/manifest.yaml:24]:
```suggestion
    description: "Whether the sub string source (SUB_STR_SRC) is a regular expression or not."
```

File: steps/github/get-file/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+kind: Step
+metadata:
+  name: "github_get_file"
+  description: "Query GitHub to retrieve a specified file."
+  vendor: github
+  type: triage
+
+envs:
+  - name: OWNER
+    description: "Owner name of the repository. Can be a user or an organization name."
```
Comment (hagaishapira) [inline steps/github/get-file/manifest.yaml:11]:
```suggestion
    description: "Owner of the repository. Can be a user or an organization name."
```

File: steps/github/get-file/manifest.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    type: "string"
+    required: true
+    example: "my-org"
+  - name: BRANCH
+    description: "Name of the remote branch in which the file resides."
+    type: "string"
+    default: "main"
+    required: true
+    example: "my-branch"
+  - name: REPOSITORY
```
Comment (hagaishapira) [inline steps/github/get-file/manifest.yaml:21]: Put REPOSITORY before BRANCH, just so it makes more sense to a user reading the docs

File: steps/github/create-pull-request/manifest.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    default: "main"
+    required: true
+    example: "my-branch"
+  - name: HEAD_BRANCH
+    description: "Name of the branch you want to eventually merge into."
+    type: "string"
+    default: "main"
+    required: true
+    example: "my-branch"
+  - name: REPOSITORY
```
Comment (hagaishapira) [inline steps/github/create-pull-request/manifest.yaml:27]: REPO before branches

**Pull request merged**